client/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
why not es6?

https://www.typescriptlang.org/tsconfig#target

Reply:
es6 less browser support. No added benefit in code language.